internal/connect/command_offset_describe.go

```diff
@@ -25,10 +26,14 @@ type humanOffsetConnectorOut struct {
 }
 
 type serializedOffsetConnectorOut struct {
-	Id       string                                      `json:"id" yaml:"id"`
-	Name     string                                      `json:"name" yaml:"name"`
-	Offsets  []map[string]any                            `json:"offsets" yaml:"offsets"`
-	Metadata connectv1.ConnectV1ConnectorOffsetsMetadata `json:"metadata" yaml:"metadata"`
+	Id       string            `json:"id" yaml:"id"`
+	Name     string            `json:"name" yaml:"name"`
+	Offsets  []map[string]any  `json:"offsets" yaml:"offsets"`
+	Metadata offsetMetadataOut `json:"metadata" yaml:"metadata"`
+}
+
+type offsetMetadataOut struct {
+	ObservedAt time.Time `json:"observed_at,omitempty" yaml:"observed_at,omitempty"`
 }
```

Comment:
Why don't we moved `ObservedAt` into `serializedOffsetConnectorOut` instead of creating this wrapper struct?

Reply:
(Are we planning on adding more metadata fields in the near future?)

Reply:
trying to keep it similar to the api structure

Reply:
Yes, more fields are to be added in future